src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,98 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private int size = 0;
+    private T[] objects;
+
+    public ArrayList() {
+        objects = (T[]) new Object[10];
+    }
+
     @Override
     public void add(T value) {
-
+        increasingTheSize();
```

Comment:
as described in common mistakes, let's rename your 'increasingTheSize()' method to make it's purpose more clear